### PR TITLE
Eslint

### PR DESCRIPTION
**Pull request by expe**
Changes to comply with new linting rules in OBT 4. Please note there are still warnings regarding the valid-jsdoc rule (for instance missing description and return).

DO NOT MERGE until OBT 4 is released.

### CODE REVIEW

File: src/javascript/core/queue.js [hunk 1]
```diff
@@ -1,8 +1,7 @@
 /*global module, require */
-'use strict';
 
-var utils = require('../utils');
-var Store = require('./store');
+const utils = require("../utils");
```
Comment (AlbertoElias) [inline src/javascript/core/queue.js]: Why change the quotes?

File: src/javascript/core/send.js [hunk 2]
```diff
@@ -1,29 +1,28 @@
...
-var Queue = require('./queue');
+const settings = require("./settings");
+const utils = require("../utils");
+const Queue = require("./queue");
 
 /**
  * Default collection server.
  */
-var domain = 'http://test.spoor-api.ft.com';
+let domain = "http://test.spoor-api.ft.com";
```
Comment (AlbertoElias) [inline src/javascript/core/send.js]: quotes?

File: src/javascript/core/session.js [hunk 3]
```diff
@@ -1,15 +1,14 @@
...
+let store;
+const defaultSessionConfig = {
 	storage: 'best',
 	name: 'session',
 	expires: (30 * 60 * 1000) // 30 minutes
 };
 
-var utils = require('../utils');
-var Store = require('./store');
+const utils = require("../utils");
```
Comment (AlbertoElias) [inline src/javascript/core/session.js]: quotes?

File: src/javascript/core/store.js [hunk 4]
```diff
@@ -9,20 +8,19 @@
...
  * @param {Object} config Optional, config object for extra configuration
  */
-var Store = function (name, config) {
+const Store = function (name, config) {
 
 	/**
 	 * Internal Storage key prefix.
 	 */
-	var keyPrefix = 'o-tracking';
+	const keyPrefix = "o-tracking";
```
Comment (AlbertoElias) [inline src/javascript/core/store.js]: quotes?

File: src/javascript/core/store.js [hunk 5]
```diff
@@ -9,20 +8,19 @@
...
 	/**
-	 * Temporary var containing data from a previously saved store.
+	 * Temporary let containing data from a previously saved store.
 	 * @property loadStore
 	 */
-	var loadStore;
-
-	var utils = require('../utils');
+	let loadStore;
+	const utils = require("../utils");
```
Comment (AlbertoElias) [inline src/javascript/core/store.js]: quotes?

File: src/javascript/core/user.js [hunk 6]
```diff
@@ -1,17 +1,16 @@
...
 	storage: 'cookie',
 	name: 'spoor-id',
 	value: null,
 	domain: (document.URL.match(/^https?\:\/\/([^\/?#]+)(?:[\/?#]|$)/i)[1].indexOf('ft.com') > -1 ? 'ft.com' : null)
 };
 
-var utils = require('../utils');
-var Store = require('./store');
+const utils = require("../utils");
+const Store = require("./store");
```
Comment (AlbertoElias) [inline src/javascript/core/user.js]: quotes?

File: src/javascript/link.js [hunk 7]
```diff
@@ -1,19 +1,19 @@
 /*global module, require, window */
-'use strict';
 
-var Queue = require('./core/queue');
-var Core = require('./core');
+const Queue = require("./core/queue");
+const Core = require("./core");
 
-var utils = require('./utils');
+const utils = require("./utils");
```
Comment (AlbertoElias) [inline src/javascript/link.js]: quotes?

File: src/javascript/page.js [hunk 8]
```diff
@@ -1,15 +1,14 @@
 /*global module, require */
-'use strict';
+const Core = require("./core");
+const utils = require("./utils");
```
Comment (AlbertoElias) [inline src/javascript/page.js]: quotes

File: src/javascript/utils.js [hunk 9]
```diff
@@ -1,16 +1,15 @@
 /*global module, require, window */
-'use strict';
 
 /**
  * Shared 'internal' scope.
  * @private
  */
-var settings = require('./core/settings');
+const settings = require("./core/settings");
```
Comment (AlbertoElias) [inline src/javascript/utils.js]: quotes?

File: test/core/queue.test.js [hunk 10]
```diff
@@ -1,9 +1,8 @@
 /*global require, describe, it */
-"use strict";
 
-var assert = require('assert'),
-	Queue = require("../../src/javascript/core/queue"),
-	queue_name = 'queue_test';
+const assert = require('assert');
+const Queue = require("../../src/javascript/core/queue");
```
Comment (AlbertoElias) [inline test/core/queue.test.js]: double quotes being used here

File: test/core/settings.test.js [hunk 11]
```diff
@@ -1,8 +1,7 @@
 /*global require, describe, it */
-"use strict";
 
-var assert = require('assert');
-var Settings = require("../../src/javascript/core/settings");
+const assert = require('assert');
+const Settings = require("../../src/javascript/core/settings");
```
Comment (AlbertoElias) [inline test/core/settings.test.js]: quotes

**Comment by AlbertoElias** [comment]
did the linter suggest that you use double quotes? There are quite a lot of quote changes

**Comment by expe** [comment]
My bad, looks like I merged with a newer version which changed to single quotes by I missed the changes. Interesting that my linter didn't complain though. Will look in to this.

**Comment by AlbertoElias** [comment]
Your linter isn't supposed to complain. Do try the latest obt beta version though. And regarding quotes, if they're single quotes, just leave as is

**Comment by expe** [comment]
Thanks, have changed the unnecessary double quote modifications back to single quotes.

**Pull request merged**